c_api/include/taichi/taichi_vulkan.h

```diff
@@ -1,6 +1,5 @@
 #pragma once
 #include <taichi/taichi_core.h>
-#include <vulkan/vulkan.h>
```

Comment:
Are we gonna get sth like "unkown dtype VkInstance"?

Reply:
Yes, so instead of striping off the includes, we allow the user to decide whether to use these 3rd party includes.